### DISCUSSION TITLE
App/Mobile Strategy

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#5495] [Archive]
Hi -
I am currently not sure whether I fully understand the use cases of Directus in the long-term.

Key question for me is:  do you intend to use the Directus App also for End Users?  Should it also be able to cover Mobile PWA functionality for end users?  Or is this a use case which should not be considered as part of Directus App and would require separate development / module / extension / ....

Thanks for any clarification you can provide.

**Comment by rijkvanzanten** [comment]
> do you intend to use the Directus App also for End Users?

Who do you consider _End Users_ in your question? The end user can be a wildly different group of people based on your use case 🙂 

> Should it also be able to cover Mobile PWA functionality for end users?

The Directus admin app can be used as a mobile PWA, yes, though we won't be implementing any offline capabilities (as you'd want to make sure the data integrity is maintained (eg data is up to date and refreshed live))

**Reply by benhaynes** [reply]
To piggyback on this... if your _end users_ are authorized/internal/allow-listed staff, then yes, that is the current use-case for Directus. In this example, you can think of Directus as a CMS, DAM, Inventory, BI, whatever tool.

If you're saying the "End User" is a public/unknown user that finds the site, registers, and uses the platform... that is not something we are aiming to support **now**, but are looking into in future versions. You can see Feature Requests for Public/SSO User Registration, Public Pages/Forms, etc that relate to this. 

As Rijk said, it really depends on how you're trying to use Directus... which we remain relatively unopinionated on.

**Reply by ernst86913** [reply]
Great - thanks a lot for giving me some insights on that.
